lib/openstudio/workflow/util/measure.rb

```diff
@@ -342,6 +342,8 @@ def apply_measure(registry, step, options = {}, energyplus_output_requests = fal
             result = nil
             begin
               load measure_path.to_s
+              # load.c in ruby can result in changing dir to root / so preserve cwd here. happens in openstudio cli 
+              Dir.chdir measure_run_dir 
```

Comment:
When doing this I prefer to always change back to the previous dir. Something like below (pseudo-ish code)

```ruby
cur_dir = Dir.pwd
begin
  Dir.chdir measure_run_dir
  <do something>>
ensure
  Dir.chdir cur_dir
end

Reply:
yeah, we certainly do that. The `measure_run_dir` is set in the same block above so I was trying to minimize the code for this.